_posts/2018-09-12-september-office-hours.md

```diff
@@ -0,0 +1,72 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-11
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+author_staff_member: burton
+dark_background: true
+---
+
+The OpenFaaS community holds regular office hours for anyone to join in as well as specialized discussions with project members, and core contributors. During these discussions, we talk about what's new in the project, where we're headed, and any other updates individual members may have.
```

Comment:
Mention the three types of calls and their purpose (covered in video) and roughly how many people that corresponds to.

Reply:
I commented elsewhere on that, I think that could be better in its own post about community and what it takes to run/build/maintain a vibrant project.